mkdocs/search.py

```diff
@@ -90,7 +100,7 @@ def generate_search_index(self):
         page_dicts = {
             'docs': self._entries,
         }
-        return json.dumps(page_dicts, sort_keys=True, indent=4)
+        return json.dumps(page_dicts, sort_keys=True)
```

Comment:
See the compact encoding in the docs, we should consider doing that:

    json.dumps([1,2,3,{'4': 5, '6': 7}], separators=(',',':'))

From https://docs.python.org/2/library/json.html

I don't really know if it has any drawbacks or limitations.